test/OpenTelemetry.Tests/Implementation/Configuration/MeterFactoryTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// <copyright file="CounterCleanUpTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using System.Linq;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Metrics.Configuration;
+using OpenTelemetry.Metrics.Export;
+using OpenTelemetry.Metrics.Test;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Configuration.Tests
+{
+    public class MeterFactoryTests
+    {
+        [Fact]
+        public void DefaultMeterShouldBeCollectedAsWell()
```

Comment:
while this test does test the default meter scenario, i was wondering if it makes sense to test just the factory alone. Something like checking if default meter, and any other meters are stored in its meterRegistry (which could be marked internal to test)

Thoughts?

(I fully understand a lot of testing in metrics codebase is already doing indirect testing, instead of proper unittest :) )

Reply:
I agree meter factory testing would be valuable here :+1:

Reply:
Yeah, this test can be added to `PushControllerTest.cs` if `PushMetricController.meters` being marked as internal, and if you prefer this way, I'll do the modification and update the branch.